tests/test_kitfile_output.py

```diff
@@ -48,6 +57,14 @@ def test_everything(fixtures: dict[str, str]):
     print("=======================================================")
 
     kitfile.manifestVersion = "3.0"
+    kitfile.package = Package.model_validate(
```

Comment:
I wasn't super clear early on this, which is my fault and these tests aren't the verbose. I went and update them in this pr: https://github.com/kitops-ml/pykitops/pull/72/files#diff-5c16520ae8f6c95d7a3ef631f6cc1d1d3a6aaf5bcc2fda0c38a13398269fb937R90. Can you add similar ones here but that cover the `kitfile.package = Package.model_valudate(...)` case? I would expect to assign to the `kitfile.package` (like you have here) and then validate those changes were updated.